recipes/french-cities/meta.yaml

```diff
@@ -1,5 +1,6 @@
 {% set name = "french-cities" %}
 {% set version = "1.1.1" %}
+{% set python_min = "3.9" %}
```

Comment:
You only need to set this one if the package min is higher than conda-forge's min.

```suggestion
```